src/prog_models/prognostics_model.py

```diff
@@ -32,6 +32,9 @@ def __init__(self, model, dict_in = {}, callbacks = {}):
             # Deepcopy is needed here to force copying when value is an object (e.g., dict)
             self[key] = deepcopy(value)
 
+        # Add class variables for length of inputs, outputs, events, state, and performance
+        self.lenStates = len(self.states)
```

Comment:
```suggestion
        self.n_states = len(self.states)
```
Just a preference thing- I prefer the notation n_ instead of len...

Also note that python naming conventions use underscores for variable names.